src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/dwplapse/DwpLapseCaseHandlerTest.java

```diff
@@ -80,10 +84,27 @@ public void givenDwpLapseCaseEventWithEmptyDwpDocuments_thenSetInterlocReviewToA
 
         assertNull(response.getData().getDwpLT203());
         assertNull(response.getData().getDwpLapseLetter());
-        assertEquals("lt203Link", response.getData().getDwpDocuments().get(0).getValue().getDocumentLink().getDocumentUrl());
-        assertEquals(DwpDocumentType.DWP_LT_203.getValue(), response.getData().getDwpDocuments().get(0).getValue().getDocumentType());
-        assertEquals("lapseLink", response.getData().getDwpDocuments().get(1).getValue().getDocumentLink().getDocumentUrl());
-        assertEquals(DwpDocumentType.DWP_LAPSE_LETTER.getValue(), response.getData().getDwpDocuments().get(1).getValue().getDocumentType());
+
+        assertEquals(2, response.getData().getDwpDocuments().size());
+
+        assertThat(response.getData().getDwpDocuments(), hasItem(
+                hasProperty("value",allOf(
```

Comment:
Very minor can you fix casing so that there is a space after , - for all of these tests